core/deployment/src/main/java/io/quarkus/deployment/console/ConsoleStateManager.java

```diff
@@ -126,6 +126,7 @@ void installBuiltins(DevModeType devModeType) {
                                 ? toLevel(((LogManager) LogManager.getLogManager()).getLogger("").getLevel()).toString()
                                 : currentLevel.toString())),
                 ConsoleStateManager.this::toggleLogLevel));
+        commands.add(new ConsoleCommand((char) 12, null, null, 10002, null, this::clearScreen));
```

Comment:
Just not sure if `12` is the right character here

Reply:
It's the ASCII form-feed character, which is also known as Ctrl-L.

(L is also the twelfth letter in the alphabet, not coincidentally)